.github/workflows/build-extension.yml

```diff
@@ -1,5 +1,5 @@
 name: Build Extension
-on: [push]
+on: [push, pull_request]
```

Comment:
I've tried this before and found the behaviour quite strange.

As it's already push, doesn't it run twice for a PR: both PR & the branch push?